horovod/tensorflow/__init__.py

```diff
@@ -103,6 +103,17 @@ def broadcast_global_variables(root_rank):
     return tf.group(*[tf.assign(var, broadcast(var, root_rank))
                       for var in tf.global_variables()])
 
+def broadcast_variables(root_rank, variables):
```

Comment:
Since we're introducing this new function, let's reimplement `broadcast_global_variables` using it.

Reply:
thanks. i updated it .